sw/host/spiflash/verilator_spi_interface.cc

```diff
@@ -61,7 +62,7 @@ int OpenDevice(const std::string &filename) {
 int ReadBytes(int fd, uint8_t *rx, size_t size) {
   size_t bytes_read = 0;
   while (bytes_read != size) {
-    size_t read_size = read(fd, &rx[bytes_read], size - bytes_read);
+    int read_size = read(fd, &rx[bytes_read], size - bytes_read);
```

Comment:
nit: `ssize_t` should be the right data type here and also for the `ReadBytes()` function return type.